chess-AI/train.py

```diff
@@ -80,126 +100,234 @@ def training_game(self):
             if board.is_game_over() or board.can_claim_draw():
                 break
 
+        # valu_approximator will be none for mcts
         if self.val_approximator is None:
-            state_values = self.assign_rewards(board, len(all_move_probs))
+            all_state_values = self.assign_rewards(board, len(all_move_probs))
 
-        return board_fens, state_values, all_move_probs
+        return board_fens, all_state_values, all_move_probs
 
     def assign_rewards(self, board, length):
         """Iterates through training examples and assigns rewards based on result of the game.
+        NOTE: Want to triple check this logic, reward assignment is very important
         """
         reward = 0
-        values = [0 for _ in range(length)]
+        
+        state_values = [0 for _ in range(length)]
 
+        # If board.outcome() is none , then there is a draw
         if (board.outcome() is not None) and (board.outcome().winner is not None):
             reward = -1
         for move_num in range(length - 1, -1, -1):
             reward *= -1
-            values[move_num] = reward
+            state_values[move_num] = reward
+
+        return state_values
 
-        # For demonstration print the board and outcome
-        # print(board)
-        print(values)
-        print(board.outcome())
-        return values
+    def create_dataset(self, games):
+        """Builds a dataset with the size of (games)
+
+        Parameters:
```

Comment:
Should be `Attributes` instead of `Parameters` for consistency with the other code, and `games` should be indented.